sirepo/job_supervisor.py

```diff
@@ -271,6 +272,7 @@ def __db_init(self, req, prev_db=None):
         c = req.content
         self.db = PKDict(
             alert=None,
+            cancelledDueToTimeout=False,
```

Comment:
`cancelledAfterSecs=0`.  Then, when the timeout happens, fill in the number of seconds, so that the error message in the GUI can be more informative.

Add to history.